### PR TITLE
TE powerbackpack

### PR DESCRIPTION
**Pull request by 00ze-cyclone**
## About The Pull Request

Add the TE powerback, just like a TE pouch but it goes on your back. No linking with guns, no self charge, just a lot of power.
Contains 20 TE cells worth of power, can be found in clothes vendor and weapon vendor.
lasbackpack
lasbackpack

## Why It's Good For The Game
Lasers don't have ammo boxes like all ther other guns, this is an equivalent to the big ammo boxes you can wear on your back.
I could have just made an ammo box counterpar for lasers but I preferred the idea of the ammo storage being batteries that go in equipment slots rather than items you can put in any storage.
I choose 20 cells worth of power because that's the amount of MG42 mags you can refill with a big 10x24mm ammo box.

## Changelog

:cl:
add: TE powerbackpack: 20 cells worth of power on your back
/:cl:

### CODE REVIEW

**Comment by Kuro020** [comment]
Worth a TM, if nothing else

**Review by Kuro020 — APPROVED** [review]

**Comment by Looking4bros** [comment]
it's Joever...

**Comment by 00ze-cyclone** [comment]
I'll reopen it once we have the server back online

File: code/modules/projectiles/magazines/energy.dm [hunk 1]
```diff
@@ -205,3 +205,13 @@
 	maxcharge = 2400
 	cell_type = /obj/item/cell/lasgun/lasrifle
 	self_recharge = FALSE
+
+/obj/item/cell/lasgun/volkite/powerpack/marine/backpack
+
+	name = "\improper TE powerbackpack"
```
Comment (TiviPlus) [inline code/modules/projectiles/magazines/energy.dm:211]:
```suggestion
	name = "\improper TE powerbackpack"
```

**Pull request merged**